src/applications/static-pages/widgetTypes.js

```diff
@@ -15,4 +15,5 @@ export default {
   VIEW_DEPENDENTS_CTA: 'view-dependents-CTA',
   FORM_686_CONTENT_REVEAL: '686UnauthContentBlock',
   CAREGIVER_CONTENT_TOGGLE: 'caregiver-content-toggle',
+  PRESCRIPTIONS_REFILL_CONTENT: 'prescriptions-refill-content',
```

Comment:
This will need to be registered in vets-api as well to appear as a Flipper option

Reply:
Oh awesome! Do you have a code snippet/PR/etc. you share how that's been done elsewhere for another widget?

Reply:
it's this file, https://github.com/department-of-veterans-affairs/vets-api/blob/master/config/features.yml